src/cli/shared/Cicshlq.option.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright IBM Corp, 2019
+*
+*/
+
+import { ICommandOptionDefinition } from "@brightside/imperative";
+
+const MAX_LENGTH = 35;
+
+/**
+ * Imperative option for the "cicshlq" parameter
+ *
+ */
+export const CicshlqOption: ICommandOptionDefinition = {
+    name: "cicshlq",
+    aliases: ["hlq"],
+    type: "string",
+    required: true,
+    stringLengthRange: [1, MAX_LENGTH],
+    description: "Specifies the High Level Qualifier (up to 35 characters) at which the CICS installation " +
```

Comment:
> CICS installation datasets

Is this the right term here? Sounds to me like the datasets that you install CICS from.  Suggest 'CICS datasets'

Reply:
I actually started out with "CICS datasets" but thought that was too vague! I'd hoped to distinguish between the CICS installation image and any region specific configuration that is created post installation (trace file, csd, etc). I looked for standard terminology around hlqs, and found several different ways of describing the concept, none of which seemed especially good. Anyway, happy to switch it back.